_posts/en/newsletters/2019-09-18-newsletter.md

```diff
@@ -0,0 +1,272 @@
+  of the LN in their channels."
+
+    Looking at how we arrived at this hub-and-spoke type of topology,
+    Kirk-Cohen notes the attractiveness of well-known nodes in the early
+    network when all connections were made manually, then the
+    attractiveness of well-connected nodes to the early LND autopilot,
+    and finally the ongoing attractiveness of popular liquidity
+    providers for people who want to receive or route payments.
+
+    Possible improvements discussed involve leveraging various graph
+    metrics to help autopilots choose channels that are well-connected
+    but not necessarily hubs (such as nodes that are one hop away from a
+    hub).  Additionally, she has been working on a scoring system for
+    when to *close* a channel and so make your funds available for
+    opening a channel to a more effective peer.  Finally, she notes that
+    changing channel connections costs onchain fees, so it's not
+    practical to make sudden changes to the topology; any changes are
+    likely to be applied slowly over time as channels are opened and
+    closed organically, so we may have to wait a long time to see
+    what effect changes have.
+
+## Scaling Bitcoin
+
+The sixth Scaling Bitcoin conference was held in Tel Aviv, Israel,
+September 11th and 12th.  Many of the topics and discussions focused on
+subjects we've already covered at length in the newsletter (such as
+[Erlay][], [COSHV][], [signet][signet progress], [miniscript][], and
+[bitcoin vaults][]).  Of the remaining topics [transcribed][sb ts] by Bryan
+Bishop, we found the following topic particularly interesting:
+
+- **TxProbe: discovering Bitcoin's network topology using orphan transactions**
+  presented by Sergi Delgado-Segura ([video][vid txprobe],
+  [transcript][ts txprobe], [paper][paper txprobe]) describes the
+  advantages and disadvantages of being able to determine Bitcoin's
+  network topology---which nodes connect to which other
+  nodes---something Bitcoin Core tries to prevent.  Delgado-Segura then
+  discloses a novel technique he and his co-authors developed for
+  probing the network to determine its topology, a combination of using
+  orphan transactions (child transactions received before at least one
+  of their parents) and a method for temporarily delaying the relay of
+  transactions between peers.  The talk illustrates some of the
+  challenges of keeping Bitcoin secure and private at the P2P network
+  level.
+
+    Although not described in detail during the talk, but perhaps worth
+    mentioning here, are the mitigations Bitcoin Core developers have
+    deployed as a result of the paper.  The first of these was Bitcoin
+    Core [PR#14626][Bitcoin Core #14626], released as part of 0.18.0,
+    which made orphan transactions less effective for probing (i.e., it
+    fixes the problem described in section 4.3 of the TxProbe paper).
+    [PR#14897][Bitcoin Core #14897], described in [Newsletter
```

Comment:
Super nit: Sergi did talk some about 14897 during the Q&A: https://youtu.be/-gdfxNalDIc?t=13405.